conf/initializers/oauth2.go

```diff
@@ -61,10 +61,7 @@ func SetUpOauth2() {
 func internalErrorHandler(err error) (response *errors.Response) {
```

Comment:
SHouldn't the return type be removed as well since now nothing is returned by the method? 🤔

Reply:
@olivierobert Removed all unused handler ✌️